phys/module_sf_pxlsm.F

```diff
@@ -658,8 +658,9 @@ SUBROUTINE pxlsm(U3D, V3D, DZ8W, QV3D,T3D,TH3D, RHO,         &
           
           END DO                        ! Time internal PX time loop   
 
-          TSK(I,J)    = TSLB(I,1,J)     ! Skin temp set to 1 cm soil temperature in PX for now
-          CANWAT(I,J) = WR * 1000.      ! convert WR back to mm for CANWAT
+!          TSK(I,J)    = TSLB(I,1,J)     ! Skin temp set to 1 cm soil temperature in PX for now
+          IF (IFLAND .LT. 1.5) TSK(I,J)    = TSLB(I,1,J)     ! ATV 02/20: changed for compatibility with sst_skin = 1.
+	  CANWAT(I,J) = WR * 1000.      ! convert WR back to mm for CANWAT
```

Comment:
@coastwx 
Align the code block

Reply:
Dave,
We have updated the codes with very close attention to alignment and removed old commenting. I've also done some additional independent testing of each fix and Jon is reviewing those results as I type. Will resolve this asap.

Reply:
@coastwx 
Great!